src/Microsoft.ML.FastTree/GamTrainer.cs

```diff
@@ -1,13 +1,13 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
-
 using System;
```

Comment:
If you can add back the line after the declaration it would be better.

Reply:
done

---
In reply to: [244804207](https://github.com/dotnet/machinelearning/pull/1929#discussion_r244804207) [](ancestors = 244804207)